src/modules/user/services/user.metaEsdt.service.ts

```diff
@@ -119,9 +120,9 @@ export class UserMetaEsdtService {
         pagination: PaginationArgs,
         calculateUSD = true,
     ): Promise<UserFarmToken[]> {
-        const farmTokenIDs = await this.farmAbi.getAllFarmTokenIds(
-            farmsAddresses(),
-        );
+        const farmTokenIDs = await this.farmAbiFactory
+            .useAbi(Address.Zero().bech32())
```

Comment:
can we use a specific farm version instead of relying on the `default` case of the `switch` ?

Reply:
both options can work